app/uk/gov/hmrc/uknwauthcheckerapistub/tools/Helper.scala

```diff
@@ -20,6 +20,7 @@ import play.api.libs.json.{JsValue, Json}
 import uk.gov.hmrc.uknwauthcheckerapistub.models.Eoris
 
 trait Helper {
+  val validToken = "Bearer <VALID_TOKEN>"
```

Comment:
The token sent from our API to EIS/Stub will be base64 encoded.

So it'd look more like:

- Bearer PFZBTElEX1RPS0VOPg==

If we want our test token to be <VALID_TOKEN> we need to ensure @GaneshMaddipoti sets that in his ticket as well.

Reply:
Indeed agreeing on a list of valid tokens would be good for testing on both ends